code/modules/antagonists/changeling/powers/panacea.dm

```diff
@@ -35,3 +31,15 @@
 
 	SSblackbox.record_feedback("nested tally", "changeling_powers", 1, list("[name]"))
 	return TRUE
+
+/datum/action/changeling/panacea/proc/heal_tox(mob/living/user)
+	for(var/i in 1 to 10)
+		user.adjustToxLoss(-5) //Has the same healing as 20 charcoal, but happens faster
+		user.radiation = max(0, user.radiation - 70) //Same radiation healing as pentetic
+		user.adjustBrainLoss(-5)
+		user.AdjustDrunk(-12 SECONDS) //50% stronger than antihol
+		user.reagents.remove_all_type(/datum/reagent/consumable/ethanol, 10)
+		for(var/datum/reagent/R in user.reagents.reagent_list)
+			if(!R.harmless)
+				user.reagents.remove_reagent(R.id, 2)
+		sleep(2 SECONDS)
```

Comment:
IK this is rich coming from me who left vamp rejuv the way it is, but would you make this a status effect?